secret.go

```diff
@@ -35,11 +44,9 @@ func applySecret(ctx *cli.Context) error {
 		println("Secret cannot be an empty string")
 		return nil
 	}
-	if requiresFile {
-		headers = make(map[string]string)
-		headers["Content-Type"] = "multipart/form-data"
+	if authType == "" {
+		authType = NTLM
```

Comment:
should we add the following link in the comment for the readers to know what NTLM is ?
` // https://www.crowdstrike.com/cybersecurity-101/ntlm-windows-new-technology-lan-manager/`

Reply:
@Ompragash @Debanitrkl  suggestions here ?

Reply:
Yeah we can add a comment

Reply:
IMO, I don't think we need to as
1. We don't have to add an external link in the code comments (also not an official Microsoft link).
2. It'll make more sense to add somewhere in the documentation but not in the code comment section.